js/ui/html_editor/utils/templates_storage.js

```diff
@@ -0,0 +1,39 @@
+import { isDefined } from '../../../core/utils/type';
+
+export default class TemplatesStorage {
+    constructor() {
+        this._widgetsStorage = {};
```

Comment:
I don't think this naming is successful. We do not save widgets in this storage. To me even simple `_storage` is better